dmcontent/questions.py

```diff
@@ -544,6 +550,36 @@ def update_expected_values(options, parents, expected_set):
         return expected_values - selected_values_set
 
 
+class Date(Question):
+    """Class used as an interface for date data between forms, backend and summary pages."""
+
+    FIELDS = ('year', 'month', 'day')
+
+    @property
+    def is_date(self):
+        return True
+
+    def summary(self, service_data):
+        return DateSummary(self, service_data)
+
+    def get_data(self, form_data):
+        """Retreive the fields from the POST data (form_data).
+
+        The d, m, y should be in the post as 'questionName-day', questionName-month ...
+        Extract them and format as 'YYYY-MM-DD'.
```

Comment:
I guess technically we can't gurantee it will be 'YYYY-MM-DD' as the user input could be anything such as 'YYYY-M-D'. This is a really anal comment though...